Britannia/events/BTWK_frontier_events.txt

```diff
@@ -470,7 +465,31 @@ BTWK_frontier_event.0202 = {
 BTWK_frontier_event.0203 = {
 	type = character_event
 	title = BTWK_frontier_event.0203
-	desc = BTWK_frontier_event.0203_desc
+	desc = {
```

Comment:
Probably also want special desc triggers for if the character holds e_roman_britannia, or for Romanized Germanics (haven't yet decided if they should be able to contest Britannia like Romans)

Reply:
Will allow holders of e_roman_britannia to get triggered_desc for 0203.high_king ("This will surely strengthen the legitimacy of your current rule over Britannia." for your convenience. Seems fairly neutral so I don't see an issue with reusing it.)